src/decaylanguage/decay/decay.py

```diff
@@ -602,6 +602,33 @@ def ndecays(self) -> int:
         """
         return len(self.decays)
 
+    def to_string(self, arrow="->") -> str:
+        """
+        One-line string representation of the entire decay chain. Sub-decays are
+        enclosed in round parentheses.
+        """
+
+        def _str(
+            decay: dict[str, list[dict[str, float | str | list[Any]]]] | str,
+            top=False,
+        ) -> str:
+            if isinstance(decay, dict):
+                mother = list(decay.keys())[0]
```

Comment:
This is fine but I reckon it would be better to write out the sets of daughters using `DaughtersDict.to_string`. Not only it shows code reuse but also ensures the various sets are ordered and hence always reproducible no matter in which order things get written in a decay file showing effectively the same final state. I think that's better and safer in the end, especially if one things of making comparisons somewhere.